README.md

```diff
@@ -132,12 +132,15 @@ nix run nix-darwin -- switch --flake ~/.config/nix-darwin
 
 ### Step 3. Using `nix-darwin`
 
-After installing, you can run `darwin-rebuild` to apply changes to your system:
+After installation, restarting your computer will make `darwin-rebuild` available in your `PATH`. After restarting, you can apply changes in your configuration by running:
 
 ```bash
 darwin-rebuild switch --flake ~/.config/nix-darwin
 ```
 
+> NOTE: if the 'darwin-rebuild' command is not found after installation, you will need to add a path to your shell configuration (f.e. the ~/.bashrc or ~/.zshrc file).
```

Comment:
```suggestion
```

If `nix-darwin` is installed correctly then the user shouldn't have this issue

Reply:
@Enzime I understand the reason for removing it, however the inclusion of guardrails can make a program more approachable for newbies